src/dispatcher.js

```diff
@@ -140,7 +141,8 @@ export default class Dispatcher {
         }
 
         const cursor = this.selectionWatcher.getFreshSelection()
-        clipboard.paste(block, cursor, afterPaste)
+        const clipboardContent = evt.clipboardData.getData('text/html')
+        clipboard.paste(block, cursor, clipboardContent, afterPaste)
```

Comment:
Should we migrate to a synchronous api already?
It always needs to be synchronous anyways, otherwise `evt.preventDefault()` won't have any effect.

```js
const blocks = clipboard.paste(block, cursor, clipboardContent)
```

Reply:
Yes, good idea, see last commit. (`cursor` can be reassigned inside the function, so I believe it also needs to be returned together with blocks)